testing/run-scripts/run_cloud.sh

```diff
@@ -64,7 +65,17 @@ then
     fi
     BANNER=`docker exec uproxy-sshd cat /banner`
   else
-    BANNER=`hostname`
+    # DigitalOcean's metadata API can tell us the region in which
+    # the machine is located:
+    #   https://developers.digitalocean.com/documentation/metadata/#metadata-api-endpoints
+    BANNER=`curl -s -m 2 http://169.254.169.254/metadata/v1/region`
```

Comment:
So the reason this doesn't couple this script tightly to DigitalOcean (vs. other cloud providers) is that this curl command will only output nonempty string when run from a DO droplet. So if we wanted to do something similar for other cloud providers in the future, we could check for empty string here, and successively try APIs of the other cloud providers until one succeeded (or other means of detection if they don't have an API). Is that right?